domain-model-assistant/Assets/Components/Scripts/CompartmentedRectangle.cs

```diff
@@ -153,9 +150,20 @@ public Vector2 GetPosition()
         return this.transform.position;
     }
 
-    public GameObject GetPopUpMenu(){
+    public GameObject GetPopUpMenu()
+    {
         return this.popupMenu;
     }
+
+    public void SetState(State aState)
+    {
+        this.state = aState;
+    }
+
+    public State GetState()
```

Comment:
Instead of Java-style getters and setters, use the more consise `{ get; set; }` construct available in C#.

Reply:
Done